pages/api/GeneratedStory.js

```diff
@@ -0,0 +1,104 @@
+import { Configuration, OpenAIApi } from "openai";
+import { v4 as uuidv4 } from "uuid";
+
+const configuration = new Configuration({
+  organization: "org-KWKX4sUxoLRs3G6H2PQOR6oN",
+  apiKey: process.env.OPENAI_API_KEY,
+});
+const openai = new OpenAIApi(configuration);
+
+async function generateCoverImage(coverImagePrompt) {
+  //   console.log(coverImagePrompt);
```

Comment:
You can remove this line.

Reply:
Will be changed before the Lunchbreak is over 😅🙈